keygen/src/keygen.rs

```diff
@@ -353,6 +376,23 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 .collect::<Vec<_>>();
             thread::park();
         }
+        ("verify", Some(matches)) => {
+            let keypair = get_keypair_from_matches(matches)?;
+            let test_data = "test".as_bytes();
+            let signature = Signature::new(&keypair.sign(&test_data).to_bytes());
+            let pubkey = if matches.is_present("pubkey") {
+                bs58::decode(matches.value_of("pubkey").unwrap())
+                    .into_vec()
+                    .unwrap()
+            } else {
+                keypair.pubkey().to_bytes().to_vec()
+            };
+            if signature.verify(&pubkey, &test_data) {
+                println!("good!");
```

Comment:
Could maybe spruce these up a bit 😉

Reply:
yea.. any suggestions?

Reply:
How about `println!("Verification for public key - {}: Success", pubkey)` and likewise for failed?